console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/client/schedule/JobScheduleAddDialog.java

```diff
@@ -322,9 +322,14 @@ public void onFailure(Throwable cause) {
                     GwtKapuaException gwtCause = (GwtKapuaException) cause;
                     if (gwtCause.getCode().equals(GwtKapuaErrorCode.DUPLICATE_NAME)) {
                         triggerName.markInvalid(gwtCause.getMessage());
-                    } else if (gwtCause.getCode().equals(GwtKapuaErrorCode.INTERNAL_ERROR)) {
-                        cronExpression.markInvalid(cause.getMessage());
-                        retryInterval.markInvalid(cause.getMessage());
+                    } else if (gwtCause.getCode().equals(GwtKapuaErrorCode.RETRY_AND_CRON_BOTH_SELECTED)) {
+                        cronExpression.markInvalid(gwtCause.getMessage());
+                        retryInterval.markInvalid(gwtCause.getMessage());
+                    } else if (gwtCause.getCode().equals(GwtKapuaErrorCode.SAME_START_AND_DATE)) {
+                        startsOn.markInvalid(gwtCause.getMessage());
+                        startsOnTime.markInvalid(gwtCause.getMessage());
+                        endsOn.markInvalid(gwtCause.getMessage());
+                        endsOnTime.markInvalid(gwtCause.getMessage());
```

Comment:
INTERNAL_ERROR is no longer handled, it should be.
Do all the fields have to be marked with invalid and have the same message?

Reply:
I spoek with Leonardo, and all of this four fields have to be marked and to have same message.